docs/LogGuide.md

```diff
@@ -2453,3 +2461,114 @@ Parsed Log Line Examples:
+and know that combatants have moved into position (or changed model or changed heading).
+These lines give more granular information when combatants change their status.
+Please note that this is still polling memory (so timing may be racy)
+and there are some heuristics to not emit too many lines (so data may be imprecise).
+
+There are three types of this line:
+
+- Add: emits all initial fields for this combatant
+- Change: emits all fields that have changed
+- Remove: no fields, combatant is being removed
+
+Each line may contain an arbitrary number of field name / value pairs.
+
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=CombatantMemory&lang=en-US) -->
+
+#### Structure
+
+```log
+Network Log Line Structure:
+261|[timestamp]|[change]|[id]|[key1]|[value1]
+
+Parsed Log Line Structure:
+[timestamp] 261 105:[change]:[id]:[key1]:[value1]
+```
+
+#### Regexes
+
+```log
+Network Log Line Regex:
+^(?<type>261)\|(?<timestamp>[^|]*)\|(?<change>[^|]*)\|(?<id>[^|]*)\|
+
+Parsed Log Line Regex:
+(?<timestamp>^.{14}) 261 (?<type>105):(?<change>[^:]*):(?<id>[^:]*)(?:$|:)
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+261|2023-04-20T19:04:39.3810000-07:00|Add|400139C5|Type|2|TargetID|10FFFFFF|Name|Omega|MaxHP|8557964|PosX|100|PosY|90|PosZ|-5.456968E-12|Heading|-4.792213E-05|Radius|12.006|BNpcID|3D5C|CurrentMP|10000|MaxMP|10000|Level|90|BNpcNameID|1E0F|WorldID|65535|CurrentWorldID|65535|NPCTargetID|1084E23D|CastDurationMax|-3.689349E+19|e173dbd66eb7c1fe
+261|2023-04-20T19:04:41.9200000-07:00|Change|400139C5|PosX|100.1179|PosY|95.16841|PosZ|-5.456968E-12|Heading|0.08906358|eac28822c9abcde9
+261|2023-04-20T19:06:46.2900000-07:00|Remove|400139C5|09a3165588ea6b13
+
+Parsed Log Line Examples:
+[19:04:39.381] 261 105:Add:400139C5:Type:2:TargetID:10FFFFFF:Name:Omega:MaxHP:8557964:PosX:100:PosY:90:PosZ:-5.456968E-12:Heading:-4.792213E-05:Radius:12.006:BNpcID:3D5C:CurrentMP:10000:MaxMP:10000:Level:90:BNpcNameID:1E0F:WorldID:65535:CurrentWorldID:65535:NPCTargetID:1084E23D:CastDurationMax:-3.689349E+19
+[19:04:41.920] 261 105:Change:400139C5:PosX:100.1179:PosY:95.16841:PosZ:-5.456968E-12:Heading:0.08906358
+[19:06:46.290] 261 105:Remove:400139C5
+```
+
+<!-- AUTO-GENERATED-CONTENT:END (logLines:type=CombatantMemory&lang=en-US) -->
+
+<a name="line262"></a>
+
+### Line 262 (0x106): RSVData
+
+Square Enix obfuscates (among other things) ability names, status names, and messages
+in current savage and ultimate content in the game data itself.
+This is to prevent data mining.
+However, as these ability names need to be displayed by the game itself
+these ability names are sent as network data upon zoning in.
+
+These lines display the currently obfuscated abilities
+for the current zone for the current game locale.
+
+Note that `:` characters are not escaped even if the game abilities have a `:` in them.
+It is recommended to use the network log line format to parse these lines for that reason.
```

Comment:
The documentation for this line should absolutely explain that CR and LF characters are escaped to prevent issues with the network log format, and therefore need to be unescaped. There are direct examples of this in the `rsv lines.txt` file I included in the PR comments, e.g.:

```
261|2023-04-21T23:24:06.0630000-04:00|en|00000051|_rsv_35827_-1_1_0_0_S13095D61_E13095D61|Further testing is required.,\r�)�� , assist me with this evaluation.|38151741aad7fe51
```

Additionally, it should probably be stated somewhere that the lines may contain special unicode character sequences that the game client will replace with placeholder values, e.g. in my example line above that gets replaced with the player's first name I believe.

In fact, we may want to modify the replace logic in OP to just escape any non-standard character with the assumption that downstream consumers will need to unescape fully, e.g. the above line might instead read:

```
261|2023-04-21T23:24:06.0630000-04:00|en|00000051|_rsv_35827_-1_1_0_0_S13095D61_E13095D61|Further testing is required.\u0002\u0010\u0001\u0003\u0002,\u000d\u00ef\u00bf\u00bd\u0007\u0002)\u0003\u00ef\u00bf\u00bd\u0002\u0003\u00ef\u00bf\u00bd\u0002 \u0002\u0003, assist me with this evaluation.|38151741aad7fe51
```

Reply:
Added some text here.

Re: escaping. I think really it'd depend on who the consumers of these messages are in terms of what made the most sense.  I don't suspect there will be many (any??) consumers of the messages part of this (vs abilities and statuses) so it probably doesn't need too much time spent massaging it unless it starts causing problems or somebody has requests for what it should look like.